### ISSUE TITLE
maxTxPower on Conduit

### ISSUE THREAD

**Issue by Smeedy** [kersing/multitech-installer#12]
Hi there,

Could it be that by disabling the lora-network-server we also did something with the maxTxPower? Since running the installer script on a spare Conduit I lost a lot of coverage with TTN mapper as it seems. I haven't done any other physical changes with this particular setup. 

From the multitech [advanced lora config](http://www.multitech.net/developer/software/lora/conduit-mlinux-lora-communication/conduit-mlinux-advance-lora-configuration/) there are settings for manipulating the system. 

Or am I missing something differently and am I barking up the wrong tree?

great work,
keep it up,
Martijn

**Comment by kersing** [comment]
Ttnmapper uses packets received, which is not related to transmit power. Do you still have the pre conversion global and local configuration files so these can be compared to the current ones? The parameters in the advanced configuration are used to generate the configuration files for the packet forwarder, once converted you are running the TTN provided configuration which may have different settings.

**Comment by Smeedy** [comment]
Yes. Seems logical. Ok, I can check the results from an other spare Conduit I still have in the box on the shelve. I'll get back on this one after comparing configs. Thx

**Comment by mjsegurapromatix** [comment]
Hi I have the same question, how to set up Tx Power on conduit, because original lora-network-server is not running after installing  ttn-pkg-fwd
I appreciate your help

**Comment by kersing** [comment]
Tx Power is set by the back-end. When running lora-network-server you are running your own back-end which requires setting the max power. Once connected to TTN the TTN back-end takes care of that.
If you want to reduce power output, find antenna gain in the global_conf.json and set it to a positive number. Setting it to negative numbers will *not* increase transmit power beyond the value specified by TTN back-end so don't bother.

**Comment by mjsegurapromatix** [comment]
Hi Kersing, thanks so much for your immediate answer.
Could you tell me how to learn about the ttn back end?
What is the maximum TTN power? Where I can find that info?

**Comment by kersing** [comment]
Look at the github repo for TTN code (github.com/TheThingsNetwork) and documentation (on the main website thethingsnetwork.org) and the TTN forum for additional information.
